src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Commands/CommandHelpers.cs

```diff
@@ -95,7 +95,7 @@ internal static void RegisterCommandHandler(Type controlType, RoutedCommand comm
 
         // 'params' based method is private.  Call sites that use this bloat unwittingly due to implicit construction of the params array that goes into IL.
         private static void PrivateRegisterCommandHandler(Type controlType, RoutedCommand command, ExecutedRoutedEventHandler executedRoutedEventHandler,
-                                                          CanExecuteRoutedEventHandler canExecuteRoutedEventHandler, params InputGesture[] inputGestures)
+                                                          CanExecuteRoutedEventHandler canExecuteRoutedEventHandler, params ReadOnlySpan<InputGesture> inputGestures)
```

Comment:
Although it is fine here, but I observed a thing - when we pass null now, it is understood as an empty ReadOnlySpan, where as earlier, when we had InputGestures[], passing null results in inputGestures to be null.

Oh I see, you have modified the inputGestures != null check as well.

It's fine.

Reply:
Yeah, as Spans are value types, passing `null` is same as an empty span; unless a logic strictly depends on the distinction between `null` and an `T[0]` array, it is always correct to take that route. Since `for` does the length checking (and `IsEmpty` is nothing but `Length == 0`, the behavior is identical).